### PR TITLE
Documentation for generating recommendations.

### PR DESCRIPTION
**Pull request by vansika**
# Problem
No docs for sending requests to generate recommendations

# Solution
Write docs about the steps involved in generating recommendations.

### CODE REVIEW

**Comment by paramsingh** [comment]
Thank you!

File: listenbrainz_spark/recommendations/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+Recommendation engine
+=====================
+ListenBrainz recommendation engine using collaborative filtering.
```
[the remark this answers is not in the document]
Comment (vansika) [reply listenbrainz_spark/recommendations/README.md:3]: Makes sense.

File: listenbrainz_spark/recommendations/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Production environemnt
+
+These instructions help to request recommendations from spark cluster in prod.
+
+The recommendation generation process has been divided into four stages.
+
+- **Stage 1**: Process user listening history.
+
+  Send a request to spark cluster for generating dataframes.
```
[the remark this answers is not in the document]
Comment (vansika) [reply listenbrainz_spark/recommendations/README.md:20]: Right!

File: listenbrainz_spark/recommendations/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+  `./develop.sh manage spark request_dataframes --days=X`
+
+  where X is the number of days of listening history you want to train the model on. By default, X is equal to 180 i.e. last 6 months from the date on which the script is invoked.
+
+- **Stage 2**: Train the model.
```
[the remark this answers is not in the document]
Comment (vansika) [reply listenbrainz_spark/recommendations/README.md:26]: These are the [params](https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz_spark/config.py.sample#L19) we are using to train the model. Rn, they are not being sent with the request and are fetched from config.py. 

But the idea sounds good to me. I can open a separate PR to send these params with spark request and then change the doc accordingly.

[the remark this answers is not in the document]
Comment (mayhem) [reply listenbrainz_spark/recommendations/README.md:26]: Please do that!

**Comment by vansika** [comment]
@paramsingh ready for merge.

**Pull request merged**